refinery/core/views.py

```diff
@@ -564,6 +563,7 @@ def solr_select(request, core):
     data = request.GET.urlencode()
     try:
         full_response = requests.get(url, params=data)
+        response_dict = json.loads(full_response.content)
```

Comment:
Missing error handling for JSON decode errors.

Reply:
in further inquiry, "is this code even used anymore?" I'll write an issue up.

Reply:
Ref #2567

Reply:
- [ ] Fix JSON decode error. We need to retain this method until we update data set search feature